src/views/tickets/tpp.php

```diff
@@ -121,18 +126,18 @@ class="tribe-tpp-message tribe-tpp-message-error tribe-tpp-message-confirmation-
 						<input
 							type="number"
 							class="tribe-tickets-quantity qty"
+							step="1"
 							min="0"
-							<?php if ( -1 !== $available ) : ?>
-								max="<?php echo esc_attr( $available ); ?>"
-							<?php endif; ?>
+							max="<?php echo esc_attr( $max_at_a_time ); ?>"
 							name="quantity_<?php echo absint( $ticket->ID ); ?>"
 							value="0"
 							<?php disabled( $must_login ); ?>
 						>
-						<?php if ( -1 !== $available && ( 0 === $threshold || $available <= $threshold ) ) : ?>
+						<?php $readable_amount = tribe_tickets_get_readable_amount( $available, null, false );
+
+						if ( - 1 !== $available && ( 0 === $threshold || $available <= $threshold ) ) : ?>
```

Comment:
:no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 341d38e178f072fd6d225dce47e4b6e3</sub>